app/views/admin/tracks/index.html.haml

```diff
@@ -6,59 +6,61 @@
         Categorize events in your conference
 .row
   .col-md-12
-    %table.table.table-hover#tracks
+    %table.table.table-hover.table-striped.table-bordered.datatable#tracks
       %thead
         %th Name
-        %th Short name
         %th Description
+        %th Room
+        %th Start Date
+        %th End Date
         %th Submitter
-        %th Color
+        %th Included in Cfp
         %th State
-        %th Included in the Cfp
         %th Actions
       %tbody
         - @tracks.each do |track|
           %tr
-            %td
-              = link_to(admin_conference_program_track_path(@conference.short_title, track)) do
-                = track.name
-            %td
-              = track.short_name
+            %td{style: "padding: 15px 0px 0px 10px;"}
+              = link_to admin_conference_program_track_path(@conference.short_title, track), class: 'btn' do
+                %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
+                  = track.name
             %td
               %p
-                = truncate(track.description)
+                = markdown(truncate(track.description))
             %td
-              - if track.self_organized?
-                = link_to track.submitter.name, admin_user_path(track.submitter)
-              - else
-                N/A
+              = track.room.try(:name)
             %td
-              %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
-                = track.color
+              = track.start_date.strftime('%A, %B %-d. %Y') if track.start_date
             %td
-              - if track.self_organized?
-                = track.state
-              - else
-                N/A
+              = track.end_date.strftime('%A, %B %-d. %Y') if track.end_date
             %td
+              = link_to track.submitter.name, admin_user_path(track.submitter) if track.self_organized?
+            %td.text-center
+              = check_box_tag "#{@conference.short_title}_#{track.short_name}", track.id, track.cfp_active,
+                class: 'switch-checkbox', method: :patch,
+                url: toggle_cfp_inclusion_admin_conference_program_track_path(@conference.short_title, id: track.short_name)+"?included=",
```

Comment:
When changing this, we have no indication about it, ie there is no flash message letting the users know if this happened, successfully or not.